tests/data/omnitron-iconverter.json

```diff
@@ -48,11 +48,11 @@
                     "sensor_index": "pseCurrent.2",
                     "sensor_type": "omnitron-iconverter",
                     "sensor_descr": "PSE Port Ouput Current 2",
-                    "group": null,
+                    "group": "",
                     "sensor_divisor": 1000,
                     "sensor_multiplier": 1,
-                    "sensor_current": 0.07678,
-                    "sensor_limit": 0.11517,
+                    "sensor_current": 0.076780000000000001,
```

Comment:
:/ getting these wierd numbers again.  Darn floats.

Reply:
should I do something about it ?